libs/sdk-core/src/lib.rs

```diff
@@ -203,13 +202,178 @@ pub use breez_services::{
 };
 pub use chain::RecommendedFees;
 pub use fiat::{CurrencyInfo, FiatCurrency, LocaleOverrides, LocalizedName, Rate, Symbol};
-pub use input_parser::{
-    parse, BitcoinAddressData, InputType, LnUrlAuthRequestData, LnUrlErrorData,
-    LnUrlPayRequestData, LnUrlRequestData, LnUrlWithdrawRequestData, MetadataItem,
-};
-pub use invoice::{parse_invoice, LNInvoice, RouteHint, RouteHintHop};
-
-pub use lnurl::pay::model::*;
 pub use lsp::LspInformation;
 pub use models::*;
+pub use sdk_common::prelude::*;
 pub use swap_out::reverseswap::{ESTIMATED_CLAIM_TX_VSIZE, ESTIMATED_LOCKUP_TX_VSIZE};
+
+// === FRB mirroring
```

Comment:
Can we move all the mirroring definitions to the binding.rs? They are only used in flutter.

Reply:
I tried different ways to do this, but unfortunately it doesn't work.

FRB mirroring expects the mirrors to be found in the root namespace of the crate, so `crate::*`. If I place them in `binding`, it fails compilation. This fails even if I re-export them in `lib.rs` as `pub use binding::*`, which should make them visible in the root namespace.